### DISCUSSION TITLE
notification policies

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#53010] [Alerting]
Greetings,

I wish the following was not free hand entry but actually brings them from what I have already entered in my alert labels

![image](https://user-images.githubusercontent.com/900951/181840619-0ee9cfc3-4a96-402b-8a5f-392cb4ca3a06.png)

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
